src/main/java/org/opentestsystem/ap/ims/service/FileUploadServiceValidator.java

```diff
@@ -39,7 +39,11 @@
     private static final String BRAILLE_NAMING_PATTERN = "^([s]tim|[i]tem|[p]assage)_(\\w+)_(enu)_(" +
```

Comment:
I would be cool with in this branch fixing IAT-665.  It seems like a good opportunity to do it, but no worries if we don't.  IAT-665 is the one to improve the regex for braille - ^(stim|item|passage)(\w+)_enu([ue][xc][ntl])(_transcript)?\.(brf|prn)$